articles/how-to-create-a-dataset-for-machine-learning/index.md

```diff
@@ -0,0 +1,108 @@
+
+1. Crowdsourcing 
+
+Crowdsourcing is used to solve various tasks ranging from simple tasks such as data labeling to complex tasks involving collaborative writing. A good example of crowdsourcing's usage is in the popular [ImageNet project](http://www.image-net.org/), which gave rise to the ImageNet image classification dataset. 
+
+In machine learning, crowdsourcing is used to aid in data generation tasks. There are two main crowdsourcing platforms that one can utilize to generate new data:
+
+The [Amazon Mechanical Turk (MTurk) ](https://www.mturk.com/) is one of the earliest and most popular examples of a crowdsourcing platform. People can sign up on this platform to complete tasks and receive payments for them.
+
+[Citizen Science](https://en.wikipedia.org/wiki/Citizen_science) is also a crowdsourcing platform whereby you can engage the public in data collection, which not only helps you collect more data but also help the public learn more about the science that you are trying to do.
+
+2. Synthetic Data Generation
+
+Synthetic data is data created via a computer simulation. It is used to increase the size of our training data or introduce changes in the data that we would like our model to handle in the future. We need these large amounts of data to have enough information to train a deep learning model properly.
+Thus, synthetic data generation usually offers us a cheaper and flexible way of expanding our datasets.
+
+Generative Adversarial Networks (GANs) is an advanced technique that we can use to generate synthetic data. It involves training two contesting networks: a generator and a discriminator. The generator's role is to learn to map a latent space to a data distribution (from a dataset). The discriminator's role is to discriminate (compare) between examples from the true distribution and the generated distribution. The goal is to increase the error rate for the discriminator network to make the generator networks so good at generating samples that will fool the discriminator into thinking that the samples are from the true data distribution (the dataset). 
+
+Using GANs effectively generates synthetic videos and images that look realistic for use in different applications. It takes in existing data and creates new data that looks like your original dataset. Thus, generating more data.
+
+#### Data Augmentation
+
+Data augmentation is another method for data acquisition. The process involves augmenting existing datasets with newly-acquired external data. Some basic steps in the data augmentation process might include cropping, flipping, rotating, brightness and contrast changing of input images.
+
+This technique enhances the size and quality of training datasets enabling you to collect more data without actually going out to physically collect more data. Another advantage of data augmentation is that it makes a models generalize better to newly unseen data.    
```

Comment:
```suggestion
This technique enhances the size and quality of training datasets enabling you to collect more data without actually going out to physically collect more data. Another advantage of Data Augmentation is that, it makes models generalize better to new unseen data.    
```

Reply:
I don't think it's correct to capitalize 'Data Augmentation' in this case @srishilesh @lalith1403